docs/get-started/how-it-works.md

```diff
@@ -28,20 +26,39 @@ Finetuner looks for a `Document` with the same `finetuner_label` (positive),
 and a `Document` with a different `finetuner_label` (negative).
 The objective is to pull `Document`s which belong to the same class together,
 while pushing the `Document`s which belong to a different class away from each other.
+````
+````{tab} Cross-modal (without label)
+Finetuner works on unlabeled text image pairs.
+You can fine-tune a CLIP like model for text to images search directly without any annotations.
+It expects either a CSV file or a {class}`~docarray.array.document.DocumentArray` consisting a list of {class}`~docarray.array.document.Document` contains two chunks: an image chunk and a text chunk.
+
+During the fine-tuning, Finetuner leverages text-image pairs and jointly optimize two models (`CLIPTextEncoder` and `CLIPImageEncoder`) with respect to two classification losses: (1) given a text, find the best matching
+image and (2) given an image, find the best matching text. Then it aggregates the two losses into the `CLIPLoss`.
+At the end, the output embedding of your data from the `CLIPTextEncoder` is comparable against the `CLIPImageEncoder`.
+````
+````{tab} Uni-modal (without label, Coming Soon)
+Finetuner works on unlabeled texts or images.
+While this feature is not opened to the user at the moment.
```

Comment:
I don't know what feature is not open.

Reply:
unimodal fine-tuning without label, as written as the title of the `tab`